apps/masterbots.ai/components/routes/profile/user-card.tsx

```
@@ -0,0 +1,331 @@
+import { Separator } from '@/components/ui/separator'
+import Image from 'next/image'
+import { BookUser, BotIcon, MessageSquareHeart,   Wand2, ImagePlus, Loader } from 'lucide-react'
+import { Button } from '@/components/ui/button'
+import { User } from 'mb-genql'
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)